### PR TITLE
Add support for pgsql

### PR DESCRIPTION
**Pull request by sabmeua**
* Add support for pgsql extension (not pdo-pgsql)
* Build confirmed with Travis CI. https://travis-ci.org/sabmeua/extra-php-extensions

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ extension=/opt/bref-extra/amqp.so
 | Memcache | `${bref:extra.memcached-php-74}` | `extension=/opt/bref-extra/memcache.so` |
 | Memcached | `${bref:extra.memcached-php-74}` | `extension=/opt/bref-extra/memcached.so` |
 | Xdebug | `${bref:extra.xdebug-php-74}` | `zend_extension=/opt/bref-extra/xdebug.so` |
+| PostgreSQL | `${bref:extra.pgsql-php-74}` | `extension=/opt/bref-extra/pgsql.so` |
```
Comment (Nyholm) [inline Readme.md]: Could you move this line above Xdebug to keep them in alphabetical order?

File: .travis.yml [hunk 2]
```diff
@@ -13,6 +13,7 @@ env:
   - LAYER=gmp PHP="72 73 74"
   - LAYER=memcached PHP="72 73 74"
   - LAYER=xdebug PHP="72 73 74"
+  - LAYER=pgsql PHP="72 73 74"
```
Comment (Nyholm) [inline .travis.yml]: If you want to please my OCD you can move this line up one too =)

File: Readme.md (hunk 1 above)
Comment (sabmeua) [reply Readme.md]: Sure. I will fix it.

File: .travis.yml (hunk 2 above)
Comment (sabmeua) [reply .travis.yml]: Also fixed this.

**Comment by Nyholm** [comment]
Thank you very much

**Pull request merged**